app/js/event/event.sort.js

```diff
@@ -32,7 +32,7 @@
           getDistanceFromLatLonInKm(Number(events[i].latitude),
                                     Number(events[i].longitude),
                                     Number(coordinates.latitude),
-                                    Number(coordinates.longitude));
+                                    Number(coordinates.longitude)).toFixed(2);
```

Comment:
I use `.toFixed()` in the proximity function when add the attribute "distanceToUser".
I will adjust the css to make it looks better.